evidence/cli/command_cli.go

```diff
@@ -70,15 +70,42 @@ func validateCreateEvidenceCommonContext(ctx *components.Context) error {
 	if !ctx.IsFlagSet(predicate) || assertValueProvided(ctx, predicate) != nil {
 		return errorutils.CheckErrorf("'predicate' is a mandatory field for creating evidence: --%s", predicate)
 	}
+
 	if !ctx.IsFlagSet(predicateType) || assertValueProvided(ctx, predicateType) != nil {
 		return errorutils.CheckErrorf("'predicate-type' is a mandatory field for creating evidence: --%s", predicateType)
 	}
-	if !ctx.IsFlagSet(key) || assertValueProvided(ctx, key) != nil {
-		return errorutils.CheckErrorf("'key' is a mandatory field for creating evidence: --%s", key)
+
+	if err := ensureKeyExists(ctx, key); err != nil {
+		return err
+	}
+
+	if !ctx.IsFlagSet(keyAlias) {
+		setKeyAliasIfNeeded(ctx, keyAlias)
 	}
+
 	return nil
 }
 
+func ensureKeyExists(ctx *components.Context, key string) error {
+	if ctx.IsFlagSet(key) && assertValueProvided(ctx, key) == nil {
+		return nil
+	}
+
+	signingKeyValue, _ := getEnvVariable(evdSigningKey)
+	if signingKeyValue == "" {
+		return errorutils.CheckErrorf("'key' or EVD_KEY must be provided when creating evidence: --%s", key)
```

Comment:
I dont understand what should I do as a user.
1. Provide a param: `key`
2. Provide a environment variable under the name `EVD_KEY`

Reply:
I agree. need a better msg for the user. The user must provide: --key OR define env var EVD_SIGNING_KEY. 
do u have a better suggestion for msg?